cmake/Cuda.cmake

```diff
@@ -1,6 +1,6 @@
 # Known NVIDIA GPU achitectures Caffe2 can be compiled for.
 # This list will be used for CUDA_ARCH_NAME = All option
-set(Caffe2_known_gpu_archs "20 21(20) 30 35 50 52 60 61")
+set(Caffe2_known_gpu_archs "20 21(20) 30 35 50 52 60 61 70")
```

Comment:
Can you add `Caffe2_known_gpu_archs8` that doesn't include 70, like the variant for CUDA 7 below?